src/flint/fmpq.jl

```diff
@@ -369,6 +369,16 @@ function isless(a::QQFieldElem, b::QQFieldElem)
   return cmp(a, b) < 0
 end
 
+function Base.isapprox(x::QQFieldElem, y::QQFieldElem;
+                       atol::Real=0, rtol::Real=0,
+                       nans::Bool=false, norm::Function=abs)
+  if norm === abs && atol < 1 && rtol == 0
```

Comment:
I dont really understand the atol<1 part here. For ZZRingElem, it's clear that two values with abs tolerance <1 already have to be equal. But I don't see why we can reason the same for QQFieldElem

Reply:
This was indeed wrong. For now I removed the `isapprox` changes from this PR in the hope that the rest can be merged, I'll fix the `isapprox` in a separate PR.

(What happened: I first copied the Julia library method for `Integer` and adjusted that. Then later I discovered I also needed it for `QQFieldElem` and just copied that without thinking. It was late *sigh*. Anyway, fixed version will come later)